docs/index.md

```diff
@@ -141,6 +141,8 @@ After successfully installing Codacy on your cluster, you are now ready to perfo
     -   Creating an initial organization
     -   Inviting users to Codacy
 
-3.  As a last step we recommend that you [set up monitoring](configuration/monitoring.md) on your Codacy instance.
+3.  If you are using Amazon EKS as your Kubernetes infrastructure and you wish to set up HTTPS for Codacy, follow our [configuration instructions](configuration/tls-ingress-aws.md).
```

Comment:
Why just for EKS?

Reply:
How do I do it for microk8s?

Reply:
Please see my comment [here](https://github.com/codacy/chart/pull/471#discussion_r433247597). Its from a previous commit, so it probably collapsed

Reply:
Yes, ideally we should have something also for MicroK8s. As soon as you have time to test the solution you have in mind @h314to, I'm glad to help. :+1: